nx_parallel/utils/chunk.py

```diff
@@ -51,3 +65,27 @@ def create_iterables(G, iterator, n_cores, list_of_iterator=None):
         return chunks(list_of_iterator, num_in_chunk)
     else:
         raise ValueError("Invalid iterator type.")
+
+
+def _get_configs(configs):
+    config_dict = asdict(configs)
+    config_dict.update(config_dict["backend_params"])
+    del config_dict["backend_params"]
+    config_dict["n_jobs"] = cpu_count(config_dict["n_jobs"])
+    if "PYTEST_CURRENT_TEST" in os.environ:
+        config_dict["backend"] = "loky"
+    return config_dict
+
+
+def get_configs(config=None):
+    """Returns the current configuration settings for nx_parallel."""
+    config_dict = _get_configs(nx.config.backends.parallel)
+    if config is None:
+        return config_dict
+    elif isinstance(config, list):
+        new_config = {k: config_dict[k] for k in config if k in config_dict}
+        return new_config
```

Comment:
Is handling a list of config attribute names standard config treatment?

Reply:
I don't think so. But, it seemed helpful to me. I'll look into it more.